### PR TITLE
routes: add boleto routes to express app

### PR DESCRIPTION
**Pull request by otaviopace**
## Description

Adapt lambda handlers which were used as
routes to work on the express web server.

Add default handlers for the `/boletos` resource and
add a default handler for any other routes and methods.

This closes https://github.com/pagarme/ghostbusters/issues/10 (internal issue)

## Your checklist for this pull request

:rotating_light: Please review this items for a good pull request. :four_leaf_clover:

1. I've read the project's [Contributing Guidelines](../CONTRIBUTING.md)
1. My commits are well written and follow [pagarme/git-style-guide](https://github.com/pagarme/git-style-guide)
1. My changes are well covered by **tests** and **logs**
1. I've updated the project docs (if needed)
1. I fell safe about this implementation
1. I feel comfortable with the code I wrote, and I'm not ashamed to show it to my friends

In a good pull request, everything above is true :relaxed:

this closes https://github.com/pagarme/ghostbusters/issues/10

### CODE REVIEW

File: src/resources/boleto/index.js [hunk 1]
```diff
@@ -35,15 +34,15 @@ const configureContext = (context = {}) => {
...
 
-const create = (event, context, callback) => {
-  configureContext(context)
-  const requestId = event.headers['x-request-id'] || defaultCuidValue('req_')()
+const create = (req, res) => {
+  const requestId = req.get('x-request-id') || defaultCuidValue('req_')()
   const service = BoletoService({ requestId })
 
-  const body = JSON.parse(event.body || JSON.stringify({}))
+  const { body: reqBody } = req
```
Comment (grvcoelho) [inline src/resources/boleto/index.js]: Tem algum motivo para chamar de `reqBody`? `body` é mais convencional eu acredito.

Comment (otaviopace) [reply src/resources/boleto/index.js]: o body usado na linha do `.tap(...` que realiza o envio da resposta usa uma variável chamada `body` e o linter fica reclamando :(

Comment (grvcoelho) [reply src/resources/boleto/index.js]: Eu chamaria aquele outro `body` de `data`. Acho que faz mais sentido. Ou então usaria `req.body` em vez de `reqBody`.

Comment (grvcoelho) [reply src/resources/boleto/index.js]: O que você acha?

Comment (otaviopace) [reply src/resources/boleto/index.js]: prefiro `req.body`, tenho flashbacks de marretadas

vou arrumar e fazer o rebase

**Pull request merged**